api/src/cle/classe/classeController.ts

```diff
@@ -242,11 +249,11 @@ router.post(
       if (!referent) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND, message: "Referent not found/created." });
       if (referent === ERRORS.USER_ALREADY_REGISTERED) return res.status(409).send({ ok: false, code: ERRORS.USER_ALREADY_REGISTERED });
 
-      const cohort = await CohortModel.findOne({ name: cohortName });
+      const classeStatus = isCleClasseCohortEnabled ? (isInscriptionOpen ? STATUS_CLASSE.OPEN : STATUS_CLASSE.ASSIGNED) : STATUS_CLASSE.CREATED;
```

Comment:
```suggestion
  let classeStatus;
  if (isCleClasseCohortEnabled) {
    classeStatus = isInscriptionOpen ? STATUS_CLASSE.OPEN : STATUS_CLASSE.ASSIGNED;
  } else {
    classeStatus = STATUS_CLASSE.CREATED;
  }
```
évtiton les double ternaire c'est illisible

Reply:
ok plus clair